locations/spiders/suitsupply.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from scrapy.selector import Selector
+from scrapy.spiders import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_3_LETTERS_FROM_SUNDAY, OpeningHours
+from locations.pipelines.address_clean_up import merge_address_lines
+
+COUNTRY_DEFAULT_LANGUAGE = {
+    "AE": "en-ae",
+    "AU": "en-au",
+    "BE": "en-be",
+    "CA": "en-ca",
+    "CH": "en-ch",
+    "CN": "zh-cn",
+    "DE": "de-de",
+    "DK": "en-dk",
+    "EE": "en-ee",
+    "ES": "es-es",
+    "FI": "en-fi",
+    "FR": "fr-fr",
+    "GB": "en-gb",
+    "IT": "it-it",
+    "LT": "en-lt",
+    "LV": "en-lv",
+    "NL": "en-nl",
+    "SE": "sv-se",
+    "US": "en-us",
+}
+SEARCH_COUNTRIES = {
```

Comment:
Where did this list of countries come from? Can it be extracted using the spider instead of hardcoded?

Reply:
`SEARCH_COUNTRIES` is from the country selection drop-down on the ["all stores" store locator page](https://suitsupply.com/en-us/stores/united-states/all).

`COUNTRY_DEFAULT_LANGUAGE` came from the website, where you scroll down to the bottom and select which country-specific site to use. All of the pages are the same across all locales though. I thought it would be courteous to use the localized website for each store's country, even though `en-us` would work fine for all of them.

Both of these could be extracted from pages on the website (store country selector is a separate AJAX call, website country selector is in the HTML). I thought it was fine to hard-code the store countries since other spiders have precedent, and the website languages because it's non-critical.

Reply:
I'm ok with it as-is, but we should try to capture this kind of stuff in the future.